superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -825,16 +852,27 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
         endpoint: `/api/v1/dashboard/${dashboard.value}/tabs`,
       })
         .then(response => {
-          const { tab_tree: tabTree, all_tabs: allTabs } = response.json.result;
+          const {
+            tab_tree: tabTree,
+            all_tabs: allTabs,
+            native_filters: nativeFilters,
+          } = response.json.result;
           tabTree.push({
             title: 'All Tabs',
             // select tree only works with string value
             value: JSON.stringify(Object.keys(allTabs)),
           });
           setTabOptions(tabTree);
+          setTabNativeFilters(nativeFilters);
 
           const anchor = currentAlert?.extra?.dashboard?.anchor;
           if (anchor) {
+            setNativeFilterOptions(
```

Comment:
Do we need to handle both `setTabNativeFilters` and `setNativeFilterOptions`? It seems that nativeFiltersOptions can be extracted from the former